### PR TITLE
[Documentation][SyliusTaxationBundle] Docs improving taxation bundle

### PR DESCRIPTION
**Pull request by Mrkisha**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

**Comment by TheMadeleine** [comment]
I am really glad you've had a look into the docs. 
Although the problem is that everything you have done is already in the docs of this bundle.

If you would like to make the existing docs better you need to **edit the existing documents** instead of creating a new one.

File: docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst [hunk 1]
```diff
@@ -0,0 +1,329 @@
+Custom use of TaxationBundle outside of Sylius eComm
+====================================================
+
+This is a guide how to use ``TaxationBundle`` for other Symfony projects, out side of ``Sylius`` eCommerce.
+Exact implementation will depend on use case.
+Installation and setting up the Bundle is same as in ``Installation`` part of this Bundle's documentation.
+
+The TaxableInterface
```
Comment (TheMadeleine) [inline docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst]: Make changes here -> http://docs.sylius.org/en/latest/bundles/SyliusTaxationBundle/taxable_interface.html

File: docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+        }
+
+        public function getName()
+        {
+            return 'acme_guest';
+        }
+    }
+
+
+Configuring taxation
```
Comment (TheMadeleine) [inline docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst]: Make changes here -> http://docs.sylius.org/en/latest/bundles/SyliusTaxationBundle/configuring_taxation.html

File: docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+        // ...
+
+        // Save the product entities.
+        $manager->persist($guestAdult);
+        $manager->persist($guestMinor);
+        $manager->persist($guestChild);
+
+    }
+
+Configure calculator
```
Comment (TheMadeleine) [inline docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst]: Changes should be done here -> http://docs.sylius.org/en/latest/bundles/SyliusTaxationBundle/custom_calculators.html

**Comment by Mrkisha** [comment]
I was not sure should i improve existing docs :

- since there are some missing pieces
- order of information and pages was weird

 or should I write separate page for how I've implemented the bundle.

According to your comments I guess I'll rewrite the docs and try to enhance existing one.

**Comment by Mrkisha** [comment]
I was confused whether to improve existing doc or add new page to how I've implemented.

According to your comments, its improvement that is needed. 
I'll rewrite it.

**Pull request closed without merging**